crates/starknet-devnet/tests/test_fork.rs

```diff
@@ -631,4 +631,41 @@ mod fork_tests {
             CAIRO_1_ERC20_CONTRACT_CLASS_HASH
         );
     }
+
+    #[tokio::test]
+    async fn test_forked_devnet_new_block_has_parent_hash_of_the_origin_block() {
+        let origin_devnet = BackgroundDevnet::spawn_forkable_devnet().await.unwrap();
+
+        let origin_block_hash = origin_devnet.create_block().await.unwrap();
+
+        let forked_devnet = origin_devnet.fork().await.unwrap();
+
+        let latest_block = forked_devnet
+            .json_rpc_client
+            .get_block_with_tx_hashes(BlockId::Tag(BlockTag::Latest))
+            .await
+            .unwrap();
```

Comment:
There are methods for retrieving the latest block on `BackgroundDevnet`: `get_latest_block_with_tx_hashes` and `get_latest_block_with_txs`

Reply:
[6dd100e](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/644/commits/6dd100ee020c0524b9c8398181ab3a707f452c52)